### ISSUE TITLE
Multiple errors when validating form

### ISSUE THREAD

**Issue by Misuki-CG** [burnacid/MyBB-Form-Creator#95]
Hi ! 

As you can see on the screen below, when validating a form (which must post as the actual user into a forum categorie), there are multiple errors and I couldn't see where it breaks :/

![image](https://user-images.githubusercontent.com/23389135/42418002-d6644a40-8296-11e8-8358-657be872ff58.png)

Here is the form (text are in french) : 
`[{"name":"Bug signalement","active":"1","class":"","settings":{"allowedgidtype":"-1","limitusage":"0","pmusers":"","prefix":"0","uid":"","customsuccess":"","signature":"0","posticon":"0","width":"","labelwidth":"","allowedgid":[""],"pmgroups":[""]},"subjecttemplate":"","messagetemplate":"","allowedgidtype":-1,"fields":[{"fieldid":"1","name":"Introduction","description":"","type":"10","default":"","required":"0","order":"0","class":"","settings":{"html":"<center>Bienvenue dans le formulaire permettant le signalement d'un bug.<\/br>\r\nAvant d'\u00e9crire un rapport de bug, vous devez v\u00e9rifier qu'un autre loup <b>n'a pas d\u00e9j\u00e0 signal\u00e9 cette erreur<\/b> et vous devez v\u00e9rifier que <b>vous utilisez bien la derni\u00e8re mise \u00e0 jour du logiciel<\/b>.<\/br>\r\nRemplissez attentivement le formulaire et ajoutez le plus d'\u00e9l\u00e9ments possibles. Plus votre signalement sera d\u00e9taill\u00e9 et plus celui-ci sera r\u00e9solu rapidement.<\/center>"}},{"fieldid":"2","name":"Titre du bug","description":"Donnez un titre au bug rencontr\u00e9.","type":"1","default":"","required":"1","order":"1","class":"","settings":{"placeholder":"","maxlength":"","regex":"","regexerror":"","size":""}},{"fieldid":"3","name":"Logiciel concern\u00e9","description":"Choisissez le logiciel concern\u00e9 par le probl\u00e8me.","type":"5","default":"","required":"1","order":"2","class":"","settings":{"options":"SnowBot PC\r\nSnowBot Touch"}},{"fieldid":"4","name":"S\u00e9parateur","description":"","type":"8","default":"","required":"0","order":"3","class":"","settings":null},{"fieldid":"5","name":"Informations g\u00e9n\u00e9rales","description":"Toutes les informations s'appliquant \u00e0 la majorit\u00e9 des bugs.","type":"9","default":"","required":"0","order":"4","class":"","settings":null},{"fieldid":"6","name":"Syst\u00e8me d'exploitation","description":"Donnez le syst\u00e8me d'exploitation que vous utilisez pour SnowBot.","type":"3","default":"","required":"1","order":"5","class":"","settings":{"options":"Windows 10\r\nWindows Server 2016\r\nWindows 8.1\r\nWindows 8\r\nWindows Server 2012\r\nWindows 7\r\nWindows Vista\r\nWindows Server 2008\r\nAutre.."}},{"fieldid":"7","name":"Type de connexion","description":"S\u00e9lectionnez la m\u00e9thode de connexion que vous utilisez.","type":"3","default":"","required":"1","order":"6","class":"","settings":{"options":"MITM\r\nSocket"}},{"fieldid":"8","name":"Explications d\u00e9taill\u00e9es","description":"Apportez-le plus d'explications possibles au sujet du bug.\t","type":"15","default":"","required":"1","order":"7","class":"","settings":{"rows":""}},{"fieldid":"9","name":"Reproduction du bug","description":"Expliquez-nous comment reproduire le bug.\t","type":"2","default":"","required":"1","order":"8","class":"","settings":{"placeholder":"","maxlength":"","regex":"","regexerror":"","cols":"","rows":"","resize":"1"}},{"fieldid":"10","name":"S\u00e9parateur 2","description":"","type":"8","default":"","required":"0","order":"9","class":"","settings":null},{"fieldid":"11","name":"Pi\u00e8ce(s) jointe(s) suppl\u00e9mentaire(s) ","description":"Apportez-le plus d'\u00e9l\u00e9ments possibles pour reproduire le bug.","type":"9","default":"","required":"0","order":"10","class":"","settings":null},{"fieldid":"12","name":"Logs","description":"Entrez vos logs qui sont affich\u00e9s dans l'onglet Console de SnowBot.\r\nCachez les informations personnelles tels que le serveur ou votre nom de personnage.","type":"2","default":"","required":"1","order":"11","class":"","settings":{"placeholder":"","maxlength":"","regex":"","regexerror":"","cols":"","rows":"","resize":"1"}},{"fieldid":"13","name":"Un trajet est-il n\u00e9cessaire pour reproduire le bug ?","description":"Le bug est-il li\u00e9 \u00e0 votre trajet ou celui-ci se manifeste sur tous les trajets ?","type":"5","default":"","required":"1","order":"12","class":"","settings":{"options":"Oui\r\nNon"}},{"fieldid":"14","name":"Lien de votre trajet ","description":"Fournissez-le trajet si vous avez s\u00e9lectionn\u00e9 \"Oui\" \u00e0 la question pr\u00e9c\u00e9dente.","type":"13","default":"","required":"0","order":"13","class":"","settings":null},{"fieldid":"15","name":"Lien de l'image de votre intelligence artificielle","description":"Donnez-nous une image de votre IA si vous consid\u00e9rez que cela est n\u00e9cessaire.","type":"13","default":"","required":"0","order":"14","class":"","settings":null},{"fieldid":"16","name":"Lien de l'image en jeu","description":"Vous pouvez ajouter une image du jeu pour appuyer vos propos.","type":"13","default":"","required":"0","order":"15","class":"","settings":null},{"fieldid":"17","name":"Lien de l'image du logiciel","description":"Apportez une image du logiciel pour une meilleure compr\u00e9hension.\t","type":"13","default":"","required":"0","order":"16","class":"","settings":null},{"fieldid":"18","name":"S\u00e9parateur 3","description":"","type":"8","default":"","required":"0","order":"17","class":"","settings":null},{"fieldid":"19","name":"Conclusion","description":"Ajoutez les derni\u00e8res informations possibles.","type":"9","default":"","required":"0","order":"18","class":"","settings":null},{"fieldid":"20","name":"Informations suppl\u00e9mentaires","description":"Si vous souhaitez apporter des informations compl\u00e9mentaires, n'h\u00e9sitez pas.\t","type":"15","default":"","required":"0","order":"19","class":"","settings":{"rows":""}},{"fieldid":"21","name":"Valider","description":"","type":"11","default":"","required":"0","order":"20","class":"","settings":null}]}]`

We are on MyBB 1.8.16. 
Thanks for reading me.

**Comment by Misuki-CG** [comment]
Ok just for information, I was wondering if it was related to the Attachments fields, and it is: without these, it doesn't show any error message. Actually, it works but it doesn't post on the forum. Don't know why, I'll try to investigate.

**Comment by burnacid** [comment]
Yes Attachements seem to be bugged for some odd reason. I haven't had time to investigate sadly. For now if you can do without please do.

**Comment by Misuki-CG** [comment]
hi !

Ok I can do without it's not a problem. For the second problem, it was my bad : on the forum I wanted to post, each posts must have prefix, but I didn't give a prefix to formcreator, so it was buggy.

I really love your plugin, thank you !
